code/modules/mob/living/simple_animal/hostile/angel_statue.dm

```diff
@@ -195,18 +195,16 @@
 	return targeting
```

Comment:
We have can have the above code give living targets direction by changing this code to be
```
	var/datum/spell_targeting/aoe/targeting = new()
	targeting.range = aoe_range
	targeting.allowed_type = /mob/living
	return targeting